api/resourcegroup.go

```diff
@@ -0,0 +1,102 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/banzaicloud/pipeline/auth"
+	"github.com/banzaicloud/pipeline/cluster"
+	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+// GetResourceGroups lists resource groups by secret
+func GetResourceGroups(c *gin.Context) {
+
+	orgID := auth.GetCurrentOrganization(c.Request).ID
+	secretId := c.Query("secretId")
```

Comment:
Shouldn't this be rather a header param then query param? IMHO query params better fit passing in filters.

Reply:
sure, fixed